README.md

```diff
@@ -33,13 +41,17 @@ The following properties must be set:
   * onegini.oauth2.clientId: the client identifier of the Web client that supports OpenID Connect
   * onegini.oauth2.clientSecret: the client secret of the Web client that supports OpenID Connect
   * onegini.oauth2.issuer: the base URL of the Token Server instance
+  
+Optional properties:  
+  * id-token-encryption.enabled: boolean for enabling ID token encryption. This should match the server side configuration
 
 ___Example configuration___
 
 ```
 onegini.oauth2.clientId=openid-client
 onegini.oauth2.clientSecret=secret
 onegini.oauth2.issuer=http://localhost:7878/oauth
+id-token-encryption.enabled=true
```

Comment:
The other properties are mapped in `ApplicationProperties`. This one isn't. Do we want to add it? If so, do we want to align the property names?

Reply:
fixed and reworked a few other property related values